index.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+
+const app = express();
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const bodyParser = require("body-parser");
+const swaggerUi = require("swagger-ui-express");
+const swaggerFile = require("./swagger_output.json");
+
+app.use(bodyParser.json());
+
+const fruits = require("./controller/fruitsController");
+
+app.use("/fruits", fruits);
+
+try {
+  app.use("/doc", swaggerUi.serve, swaggerUi.setup(swaggerFile));
```

Comment:
[Live Run] The swagger doc is not working, seems missing `fruits/` path

Reply:
![image](https://user-images.githubusercontent.com/97423112/170931421-eed227a5-352f-47ca-b433-6b68ecc283a0.png)

Reply:
Resolved.